### DISCUSSION TITLE
performance improvement with and without docker

### DISCUSSION THREAD

**Discussion by praveeny1986** [microsoft/fastseq#77] [Q&A]
Hi @feihugis @JiushengChen 

we tried fastseq for t5 on docker and we see performance improvement of ~ 1.4X only.
How can we get it above 4X improvement as per the original benchmark results ?

**Comment by JiushengChen** [comment]
Hi Praveen,

Are you using our docker file? Could you show more details for reproduce? Like gpu type, command line etc. Thanks. Happy Holidays!
